spinalcordtoolbox/vertebrae/core.py

```diff
@@ -490,7 +561,7 @@ def label_segmentation(fname_seg, list_disc_z, list_disc_value, verbose=1):
             vertebral_level = 0
         else:
             # assign vertebral level (add one because iz is BELOW the disk)
-            vertebral_level = list_disc_value[ind_above_iz] + 1
+            vertebral_level = list_disc_value[ind_above_iz]
```

Comment:
This comment seems out of date now because of the change, maybe.

Reply:
Yes It will If needed. In fact this cause a slight discrepency between the "disc" label and the "vertebral level" label. I Am trying to see which is best.